scripts/bundle.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+pushd dist
```

Comment:
They work in bash. I'm explicitly using bash in the shebang. I can just use `cd dist` and `cd ..` if you'd prefer.